README.md

```diff
@@ -311,6 +323,75 @@ Keys that don't start with an underscore typically have groups of
   the state of the game (e.g., player #, ball #, progressive jackpot value,
   etc.)  Not useful for PinMAME's `.nv` files, but could be referenced with
   custom code inside PinMAME.  See the High Speed map for an example.
+- **dip_switches**: A special section detailing DIP switch options for the
+  game.  See the "DIP Switches" section below for details.
+
+### DIP Switches
+
+PinMAME stores DIP Switch values in the last six bytes of the `.nv` file.
+The first byte represents SW1 to SW8, with SW1 in bit 0.  The second byte
```

Comment:
Maybe it's better to use least significant bit instead of bit 0. Not sure what the usual terms are for this.